src/sentry/api/urls.py

```diff
@@ -2821,6 +2822,12 @@
         r"^internal/",
         include(INTERNAL_URLS),
     ),
+    # OAuth (Sentry as a provider)
+    re_path(
```

Comment:
Actually - i would pull this into `src/sentry/web/urls.py`
and maybe stick it before
```
    # Versioned API
    re_path(
        r"^api/0/",
        include("sentry.api.urls"),
    ),
    ```
    
I don't imagine this is going to be a "versioned" api anytime soon.
we should simply keep this under `api/oauth/userinfo`